src/Controls/src/Core/HybridWebView/HybridWebView.cs

```diff
@@ -80,6 +80,34 @@ public void SendRawMessage(string rawMessage)
 				});
 		}
 
+		/// <inheritdoc/>
+		public async Task InvokeJavaScriptAsync(
+			string methodName,
+			object?[]? paramValues = null,
+			JsonTypeInfo?[]? paramJsonTypeInfos = null)
```

Comment:
This is the new API that makes using it more intuitive and does not have unused parameters.

If we do not want to break ABI with .NET 9.0 GA, we can hide this and maybe the community toolkit an provide some extension methods. The existing method will work, but we have to provide an unused generic argument.